applications/serial_lte_modem/doc/PPP_AT_commands.rst

```diff
@@ -0,0 +1,95 @@
+.. _SLM_AT_PPP:
+
```

Comment:
CGEREP special handling needs to be documented.

Reply:
Why do you think so? Everything is made so that the behavior is not changed for the end user and it's all invisible.

Reply:
The events are not really stopped. Take for example the case where there is another processor for these events. This will be very hard to investigate if there is no mention of it. Same goes for the errors that are sent from the AT-command versions which you ignore. If there is no mention of special handling of these commands anywhere, it will be problematic to investigate.

Reply:
What potential other processor for these events are you thinking of?

Regarding the alternative AT command syntax, okay. My reasoning was that these should not be allowed because of how weird they are, but it's actually a deviation from the modem behavior. The code could be updated to accept those.

Reply:
Anything that subscribes to AT_MONITOR(at_notify, ANY, notification_handler) (or specific events).

Rather than update the event, I think that it would be enough to document that we have special handling for AT+CGEREP and +CGEV, in which SLM intercepts and resends the commands.

Reply:
I think that the command would be possible from modules other than SLM, not sure how likely that is. But you are correct that then we are on the developer side.

If you can get the error cases to work (or at least log indicative errors from them), then this will not necessarily need to be indicated in the documentation.